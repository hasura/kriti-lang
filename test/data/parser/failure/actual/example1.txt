ParseError "sourceName" (line 1, column 3):
unexpected NumLit "1" 1.0