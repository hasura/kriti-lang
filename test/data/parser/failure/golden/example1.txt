RenderedError {_code = ParseErrorCode, _message = "sourceName:1:4:\n  |\n1 | { 1: null }\n  |    ^\nunexpected   1\n", _span = (SourcePosition {_sourceName = "", _line = 1, _column = 1},Nothing)}